jekyll/_cci2/configuration-reference.adoc

```diff
@@ -2550,10 +2550,11 @@ workflows:
 [#using-when-in-workflows]
 ==== *Using `when` in workflows*
 
-NOTE: Using `when` or `unless` under `workflows` is supported in `version: 2.1` configuration.
+NOTE: Using `when` or `unless` under `workflows` is supported in `version: 2.1` configuration. Using `when: always` is no longer supported as of December 3,2024. You must use `when: true`. 
```

Comment:
`when: always` has never been supported, we aren't deprecating it. `always` does not mean anything special in the context of a workflow `when`, it's just a non-empty string.

The thing to do is to delete the `when` key, the default behaviour is to always run a workflow _unless_ there is a `when` (or `unless`) key that says when not to run the workflow.

Reply:
How about

> Workflows are always run unless there is a `when` or `unless` filter that prevents the workflow from being run. If you want a workflow to run in every pipeline, do not add a `when` or `unless` filter.

Reply:
@HennaAbbas do you want to make this change in this PR?